tests/SwaggerProvider.ProviderTests/v3/Swashbuckle.FileController.Tests.fs

```diff
@@ -0,0 +1,47 @@
+module Swashbuckle.v3.FileControllersTests
+
+open Expecto
+open System
+open System.IO
+open Swashbuckle.v3.ReturnControllersTests
+
+[<Tests>]
+let resourceControllersTests =
+  let text = "This is test file content"
+  let toStream (text:string) = 
+      let bytes = System.Text.Encoding.UTF8.GetBytes(text)
+      new MemoryStream(bytes)
+  let fromStream (stream:IO.Stream) = async {
+      use reader = new StreamReader(stream)
+      return! reader.ReadToEndAsync() |> Async.AwaitTask
+  }
+
+  testList "All/v3/Swashbuckle.FileControllersTests.Tests" [
+
+    testCaseAsync "Download file as IO.Stream" <| async {
+        let! stream = api.GetApiReturnFile()
+        let! actual = fromStream stream
+        Expect.stringContains actual "I am totally a file" "incorrect server response"
+    }
+
+    testCaseAsync "Send file and get it back" <| async {
+        let data = WebAPI.OperationTypes.PostApiReturnFileSingle_formData(toStream text)
+        let! stream = api.PostApiReturnFileSingle(data)
```

Comment:
I am not supper happy that we generate wrapper object around payload ...
but in OpenAPI payload described as one parameter `requestBody` that we map to provided type

```json
    "/api/ReturnFile/single": {
      "post": {
        "tags": [
          "ReturnFile"
        ],
        "requestBody": {
          "content": {
            "multipart/form-data": {
              "schema": {
                "type": "object",
                "properties": {
                  "file": {
                    "type": "string",
                    "format": "binary"
                  }
                }
              },
              "encoding": {
                "file": {
                  "style": "form"
                }
              }
            }
          }
        },
```